dcipipeline/main.py

```diff
@@ -52,7 +52,7 @@ def generate_ansible_cfg(dci_ansible_dir, config_dir):
 module_utils       = {dci_ansible_dir}/module_utils/
 callback_whitelist = dci
 callback_plugins   = {dci_ansible_dir}/callback/
-interpreter_python = /usr/bin/python2
+roles_path         = /usr/share/dci/roles/:/var/lib/dci-openshift-agent/baremetal_deploy_repo/ansible-ipi-install/roles/
```

Comment:
@ylamgarchal should we make this a settings in the pipeline file or a command line option?

Reply:
I would say a settings since we provide the pipeline file in the cli option.